force_torque_sensor_broadcaster/src/force_torque_sensor_broadcaster.cpp

```diff
@@ -41,14 +47,6 @@ controller_interface::CallbackReturn ForceTorqueSensorBroadcaster::on_init()
     return controller_interface::CallbackReturn::ERROR;
   }
 
-  return controller_interface::CallbackReturn::SUCCESS;
-}
-
-controller_interface::CallbackReturn ForceTorqueSensorBroadcaster::on_configure(
-  const rclcpp_lifecycle::State & /*previous_state*/)
-{
-  params_ = param_listener_->get_params();
```

Comment:
where is this line got lost?

Reply:
because it's already done in the `try` statemen. See [L42](https://github.com/ros-controls/ros2_controllers/pull/698/files#diff-782354977c4a94dd7c757e1396ba30acc394ebc4de3c2c1f2adafc522eea2352R42)

Reply:
Why not? If the `frame_id ` parameter is not defined, then an exception is thrown